src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -251,6 +274,37 @@ function ExamSettings(props) {
     );
   }
 
+  function renderSaveSuccess() {
+    const studioCourseRunURL = StudioApiService.getStudioCourseRunUrl(props.courseId);
+    return (
+      <Alert
+        variant="success"
+        dismissible
+        data-test-id="saveSuccess"
+        onClose={() => setSaveSuccess(false)}
+      >
+        Proctored exam settings saved successfully.
+        You can go back to your course in Studio <Alert.Link href={studioCourseRunURL}>here</Alert.Link>.
+      </Alert>
+    );
+  }
+
+  function renderSaveError() {
+    return (
+      <Alert
+        variant="danger"
+        dismissible
+        data-test-id="saveError"
+        onClose={() => setSaveError(false)}
+      >
+        We encountered a technical error while trying to save proctored exam settings.
+        This might be a temporary issue, so please try again in a few minutes.
+        If the problem persists,
+        please go to <Alert.Link href="https://support.edx.org/hc/en-us">edX Support Page</Alert.Link> for help.
```

Comment:
Nit: Could you change the other link in `renderConnectionError` to an `Alert.Link` component?  I don't think it's a big deal; they are just styled differently, so I noticed it visually.